scripts/create.sh

```diff
@@ -0,0 +1,137 @@
+
+  echo "===================================================="
+  echo "Initializing the test application. Please be patient, it takes a few \
+  minutes to complete."
+  echo "===================================================="
+
+  # The outer loop checks for the external IP address associated with the
+  # HTTP load balancer created by the Kubernetes Engine prime-server ingress.  It
+  # should be created relatively quickly so the loop sleeps for 10 seconds and
+  # error times out after 100 seconds.
+  ELAPSED=0
+  SLEEP=10
+  MAX_TIME=100
+  while true; do
+
+    IP_ADDRESS=$(kubectl get ingress prime-server --namespace \
+      default -o jsonpath='{.status.loadBalancer.ingress..ip}')
+    if [[ -n "$IP_ADDRESS" ]]; then
+      SERVER_ADDRESS="http://${IP_ADDRESS}"
+      echo "$SERVER_ADDRESS provisioned!"
+
+      # This inner loop is to wait for the server to respond to a health check.
+      # This can take much longer so we animate the cursor to ensure that the
+      # user can see that the script has not timed out.  We do not have an error
+      # timeout for acquiring a health check because sometimes this step takes
+      # unusually long.
+      while true; do
+        echo -ne 'waiting for endpoint to become healthy: |\r'
+        sleep 1
+        echo -ne 'waiting for endpoint to become healthy: /\r'
+        sleep 1
+        echo -ne 'waiting for endpoint to become healthy: -\r'
+        sleep 1
+        echo -ne 'waiting for endpoint to become healthy: \\\r'
+        sleep 1
+        if [[ "$(curl -s -o /dev/null -w "%{http_code}" "$SERVER_ADDRESS"/)" == \
+         "200" ]]; then
+          break
+        fi
+      done
+      echo ""
+      echo "SUCCESS! $SERVER_ADDRESS is now healthy"
+      exit 0
+    fi
+    if [[ "${ELAPSED}" -gt "${MAX_TIME}" ]]; then
+      echo "ERROR: ${MAX_TIME} seconds exceeded, no response from kubernetes api."
+      exit 1
+    fi
+    echo "After ${ELAPSED} seconds, endpoint not yet provisioned, waiting..."
+    sleep "$SLEEP"
+    ELAPSED=$(( ELAPSED + SLEEP ))
+  done
+}
+
+run_build() {
+  [ -d "${ROOT}/build" ] || mkdir "${ROOT}/build"
+  tar -cvzf "$ROOT/build/flask-prime.tgz" -C "$ROOT/container" .
+
+  # Cloud Build!
+  BUILD_DATE=$(date)
+  if command -v git >/dev/null; then
+    VCS_REF=$(git rev-parse HEAD)
+  else
+    VCS_REF="NO-VCS"
+  fi
+
+  echo "Building container for prime-server version ${VERSION}"
+  gcloud builds submit "$ROOT/container" \
+    --config="${ROOT}/container/cloudbuild.yaml" \
+    --substitutions _VERSION="${VERSION}",_BUILD_DATE="${BUILD_DATE}",_VCS_REF="${VCS_REF}"
+}
+
+ROOT="$( cd "$( dirname "${BASH_SOURCE[0]}" )/.." && pwd )"
+# shellcheck source=scripts/common.sh
+source "$ROOT/scripts/common.sh"
+
+VERSION=$(cat "${ROOT}/VERSION")
+
+check_dependency_installed gcloud
+check_dependency_installed terraform
+
+# Enable required GCP services
+get_project
+enable_project_api ${PROJECT} compute.googleapis.com
```

Comment:
Need to do: `enable_project_api "${PROJECT}" container.googleapis.com`

Reply:
That's a change that should be made, but are you sure it's to blame for the validation failure? If so, can you explain for me?